packages/terra-aggregate-translations/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## Unreleased
+* Removed
```

Comment:
Should this be `Breaking`?

Reply:
Yes, because react-intl is a peer dependency and this change could break a person's build.

Reply:
updated and added upgrade guide : https://github.com/cerner/terra-toolkit/pull/739/files#diff-6cf94aac50699d6fc85ff446edc1a57df6e4384bf7b27ce1d9697887f45e1755